web-services/cached-results/src/main/java/datawave/webservice/results/cached/CachedResultsBean.java

```diff
@@ -1662,11 +1663,11 @@ public CachedResultsResponse reset(@PathParam("queryId") @Required("queryId") St
                         crq.closeConnection(log);
                     }
                     
-                    Connection connection = ds.getConnection();
-                    String logicName = crq.getQueryLogicName();
-                    QueryLogic<?> queryLogic = queryFactory.getQueryLogic(logicName, p);
-                    crq.activate(connection, queryLogic);
-                    
+                    try (Connection connection = ds.getConnection()) {
```

Comment:
I think this is another case where we would be closing the object before we're done using it.

Reply:
reverted